arch/arm/arm64/include/uk/asm/paging.h

```diff
@@ -1,9 +1,11 @@
 /* SPDX-License-Identifier: BSD-3-Clause */
 /*
  * Authors: Marc Rittinghaus <marc.rittinghaus@kit.edu>
+ *          Michalis Pappas <michalis.pappas@opensynergy.com>
  *
  * Copyright (c) 2021, Karlsruhe Institute of Technology (KIT).
  *                     All rights reserved.
+ * Copyright (c) 2022, OpenSynergy GmbH. All rights reserved.
  *
```

Comment:
Will do. I previously put yours first based on alphabetical order. Btw, the author statement is maybe obsolete these days as all information is in git. And as more people change a file, the list of authors will grow. Even if we consider only adding the original author(s), that information may also become obsolete as the code evolves, as the majority of the code may even no longer be from the original author. That is also partially the reason I created `bpt64.S` and didn't use the existing `pagetable64.S`, as it has the following note:
```
  1 /* SPDX-License-Identifier: BSD-2-Clause */
  2 /*-
  3  * This code is based on:
  4  * https://github.com/freebsd/freebsd/blob/master/sys/arm64/arm64/locore.S
  5  *
  6  * Authors: Andrew Turner
  7  *          Wei Chen <Wei.Chen@arm.com>
  8  *
  9  * Copyright (c) 2012-2014 Andrew Turner. All rights reserved.
 10  * Copyright (c) 2018 Arm Ltd. All rights reserved.
 11  *
```
After this PR we remove the majority of that code. If we added the contents of `bpt64.S` there, the statement in the header would just no longer be true.

PS: I hope you're happy with the attributions in general, as I used the x86 implementation as a reference. In `plat/common/include/arm/arm64/paging.h` I state:
```
  4  *
  5  * Based on plat/common/include/x86/paging.h by Marc Rittinghaus.
  6  *
```

Reply:
Yes, I totally agree that the authors statement is obsolete and actually, I would be fine with removing it but I am not sure if this is compatible with the license text because it says `the above copyright notice`. So something has to go there. I usually keep the authors up to a felt personal threshold where I think that so much code has changed that there is effectively no connection to the original authors. Of course, this is subjective and suboptimal.